devel/buf/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; -*-
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/bufbuild/buf 1.50.1 v
+go.offline_build    no
+revision            0
+
+categories          devel
+license             Apache-2
+maintainers         @mikaoj openmaintainer
+description         Buf is a tool for working with Protocol Buffers.
+long_description    The Buf CLI is a helpful tool for managing Protobuf schemas. \
+                    It offers various features, including code generation, breaking change detection, linting, and formatting, \
+                    to assist with Protobuf development and maintenance. \
+                    It works with your choice of plugins and languages and gives you access to a vast library of certified plugins in the Buf Schema Registry.
+homepage            https://buf.build/
+
+checksums           rmd160  be72fcd551d797aeb73f26060569c282e383e785 \
+                    sha256  2dc0e7eae6a9cc206de4421162e0f5895b9488a1614b8bf30eebd5588cd08df5 \
+                    size    1590934
+
+use_configure       no
```

Comment:
Also, the portgroup eliminates the need to specify `use_configure` and `use_build`. Both of those lines can be dropped.

Reply:
Fixed 😃